data/class/SC_FormParam.php

```diff
@@ -171,10 +171,12 @@ public function setDBDate($db_date, $year_key = 'year', $month_key = 'month', $d
         if (empty($db_date)) {
             return;
         }
-        list($y, $m, $d) = preg_split('/[- ]/', $db_date);
-        $this->setValue($year_key, $y);
-        $this->setValue($month_key, $m);
-        $this->setValue($day_key, $d);
+
+        // Smarty3以降は月日が1桁の場合、0埋めされていると補完されないため0を除去する
+        $objDate = new DateTime($db_date);
```

Comment:
細かいところですが、 DateTimeImmutable にしておきましょうか。
```suggestion
        $objDate = new DateTimeImmutable($db_date);
```

Reply:
こちら修正してPRします